tests/integration/azure_blob/test_azure_blob.py

```diff
@@ -19,15 +19,65 @@
 blob_dir = "test_mlrun_azure_blob"
 blob_file = f"file_{random.randint(0, 1000)}.txt"
 
-
-def azure_connection_configured():
-    return config["env"].get("AZURE_STORAGE_CONNECTION_STRING") is not None
+AUTH_VARS = [
+    "AZURE_STORAGE_CONNECTION_STRING",
+    "AZURE_STORAGE_ACCOUNT_NAME",
+    "AZURE_STORAGE_ACCOUNT_KEY",
+    "AZURE_STORAGE_SAS_TOKEN",
+    "AZURE_STORAGE_CLIENT_ID",
+    "AZURE_STORAGE_CLIENT_SECRET",
+    "AZURE_STORAGE_TENANT_ID",
+]
+
+AUTH_METHODS = [
+    "conn_str",
+    "sas_token",
+    "account_key",
+    "spn",
+]
 
 
-def prepare_env():
-    os.environ["AZURE_STORAGE_CONNECTION_STRING"] = config["env"].get(
-        "AZURE_STORAGE_CONNECTION_STRING"
-    )
+def azure_connection_configured():
+    for var in AUTH_VARS:
+        assert config["env"].get(var) is not None
```

Comment:
Should this be configured to assert that, at the very least, the "AZURE_STORAGE_CONNECTION_STRING" is configured?

Reply:
No. The point is that this test is opt-in, since it requires manual configuration and is designed to run manually. However, all tests under `tests/integration` are included in the integration test suite. When running the overall suite of tests (for example in the CI when doing a PR) we don't have manual configurations, so the test needs to either pass silently or be skipped. If failing the test, it will fail the test suite and for example block your PR.

Reply:
I reverted this back to the original function, as implemented, which expects AZURE_STORAGE_CONNECTION_STRING.

Are you saying you'd prefer to eliminate this function, and the `pytest.mark.skipif` calls?